.travis.yml

```diff
@@ -20,18 +20,19 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
-        - NUMPY_VERSION=1.9
+        # For now default python=3.4, since beautiful soup does not support 3.5
+        - PYTHON_VERSION=3.4
+        - NUMPY_VERSION=1.10
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautiful-soup'
         - PIP_DEPENDENCIES=''
 
     matrix:
-        - PYTHON_VERSION=2.6 SETUP_CMD='egg_info'
+        - SETUP_CMD='egg_info'
         - PYTHON_VERSION=2.7 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.3 SETUP_CMD='egg_info'
         - PYTHON_VERSION=3.4 SETUP_CMD='egg_info'
-        - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
```

Comment:
I would advocate for this matrix above always listing the PYTHON_VERSION for clarity